README.md

```diff
@@ -223,11 +224,18 @@ file for a specific client. Without the identifier collisions can happen.
 Service                                   | Driver name
 ------------------------------------------|-------------
 [Session identifier](#session-identifier) | `session`
+[Auth identifier](#auth-identifier)       | `auth`
 [NOP identifier](#nop-identifier)         | `nop`
 
 ### Session identifier
 
-This identifier uses the client session and, the original file name to create an identifier for the upload file.
+This identifier uses the client session, and the original file name to create an identifier for the upload file.
+
+### Auth identifier
+
+This identifier uses the id of the authenticated user, and the original file name to create an identifier for the upload file.
```

Comment:
You should not use a comma before _"and"_ if you’re only mentioning two qualities.

```suggestion
This identifier uses the client session and the original file name to create an identifier for the upload file.

### Auth identifier

This identifier uses the id of the authenticated user and the original file name to create an identifier for the upload file.
```